types/eth1.pb.go

```diff
@@ -2414,6 +2477,18 @@ func file_eth1_proto_init() {
 			}
 		}
 		file_eth1_proto_msgTypes[14].Exporter = func(v interface{}, i int) interface{} {
+			switch v := v.(*Eth1ERC721Indexed); i {
+			case 0:
+				return &v.state
+			case 1:
+				return &v.sizeCache
+			case 2:
+				return &v.unknownFields
+			default:
+				return nil
+			}
```

Comment:
**Question:** Do you know why this was added? From my pov, this should have been here before. Just curious.

Reply:
Oh that's weird indeed.. I have no idea honestly